### PR TITLE
root#home-design-frontend

### PR DESCRIPTION
**Pull request by nburger01**
Bildschirmfoto 2020-10-27 um 20 16 02
(In order of completion)
- Created "div class="container-fluid" (whole page container - NavBar excluded) in views/pages/home.html.erb; 
- .container-fluid in /stylesheets/pages/_home.scss (modified css properties); 
- Created search form (as a sample for future search bar) "form novalidate="novalidate" class="simple_search_form" in views/pages/home.html.erb; 
- .search-form-control, .search-form-control .btn, .search-form-control .form-control, .search-form-control .form-control:focus in /stylesheets/pages/_home.scss (modified css properties) 
- Created "div class="container-quote" (main quote on top of search bar)  in views/pages/home.html.erb; 
- .container-quote, .container-quote h1 in  /stylesheets/pages/_home.scss (modified css properties); 

MISSING IN THE HOMEPAGE: RESPONSIVE SEARCH BAR, DROP DOWN CATEGORY MENU INITIALIZED FROM NAVBAR, FOOTER

### CODE REVIEW

**Comment by Giuliamartinelli** [comment]
I love the pine tree, love everything. Except for the btn search that I think should be thicker.
Friendly reminder for navbar: Create lesson and sign-in are both bordered/colored green.

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,19 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="container-fluid">
+<h1></h1>
+
+<div class="container-quote">
+  <h1>Learning is a treasure that will follow its owner everywhere. </h1>
```
Comment (marcochellogs) [inline app/views/pages/home.html.erb:5]: > Learning is a treasure that 
   follows its master everywhere.

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -8,26 +8,31 @@
...
         <li class="nav-item active">
-          <%= link_to "Home", "#", class: "nav-link" %>
+          <%= link_to "Create a Lesson", "#", class: "nav-link" %>
         </li>
         <li class="nav-item">
-          <%= link_to "Messages", "#", class: "nav-link" %>
+          <%= link_to "Bookings", "#", class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
           <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Comment (marcochellogs) [inline app/views/shared/_navbar.html.erb:23]: make sure to replace this in the navbar

**Review by marcochellogs — COMMENTED** [review]
Nice job! 
Tomorrow we can make all the website pretty :)
Goodnight

File: app/views/shared/_navbar.html.erb (hunk 2 above)
Comment (marcochellogs) [reply app/views/shared/_navbar.html.erb:23]: giulia already changed this

**Pull request merged**